README.md

```diff
@@ -153,50 +153,41 @@ look at a 'standalone' build, which hooks Eclipse OMR up to the its testing syst
 only.
 
 ### Basic configuration and compile
-To build standalone Eclipse OMR, run the following commands from the top of the
-source tree. The top of the Eclipse OMR source tree is the directory that contains
-run_configure.mk.
+To build standalone Eclipse OMR, run the following commands from the root of the
+source tree. For more detailed instructions please read [BuildingWithCMake.md](doc/BuildingWithCMake.md).
 
-    # Generate autotools makefiles with SPEC-specific presets
-    make -f run_configure.mk SPEC=linux_x86-64 OMRGLUE=./example/glue
+    # Create a build directory and cd into it
+    mkdir build
+    cd build
+    
+    #generate the build system using cmake
+    cmake ..
 
     # Build
-    make
+    make -j
 
-    # Run tests (note that no contribution should cause new test failures in "make test")
-    make test
-
-Run `make -f run_configure.mk help` for a list of configure makefile targets.
-Run `make help` for a list of build targets.
-
-### Building Eclipse OMR on Windows
-A shell script interpreter, such as bash, is required to run configure.
-
-### How to Configure with Custom Options
-Run `./configure --help` to see the full list of configure command-line
-options.
-
-To run configure using both `SPEC` presets and custom options, pass the
-`EXTRA_CONFIGURE_ARGS` option to `run_configure.mk`.
+    # Run tests (note that no contribution should cause new test failures in testing).
+    # Use the `-V` option to see verbose output from the tests.
+    ctest [-V]
 
-For example, to disable optimizations, run configure like this:
+### Building Eclipse OMR on Windows using Visual Studio
+The following instructions below demonstrate the steps to build Eclipse OMR on Windows
+using Visual Studios. In the example Visual Studio 11 2012 Win64 is being used.
+You can easily switch this to the version of Visual Studio you would like to use.
 
-    # Example configure
-    make -f run_configure.mk SPEC=linux_x86-64 OMRGLUE=./example/glue 'EXTRA_CONFIGURE_ARGS=--disable-optimized' clean all
+    # Create a build directory and cd into it
+    mkdir build
+    cd build
+    
+    #generate the build system using cmake
+    cmake -G "Visual Studio 11 2012 Win64" ..
 
-To disable building fvtests, run configure like this:
-
-    # Example configure disabling fvtests
-    make -f run_configure.mk SPEC=linux_x86-64 OMRGLUE=./example/glue 'EXTRA_CONFIGURE_ARGS=--disable-fvtest' clean all
-
-Note that the `clean` target of `run_configure.mk` deletes the header files and
-makefiles generated by configure. Invoking the `clean all` targets ensures that
-the header files and makefiles are regenerated using the custom options.
+    # Build
+    make -j
```

Comment:
`make` will not work here since we are generating for visual studio. Instead we should be using msbuild. Or the simpler way `cmake --build .`

Reply:
fixed